docs/reports/unmapped.md

```diff
@@ -0,0 +1,19 @@
+# Mapping progress report
```

Comment:
@matentzn I had an error that was inverting my '% mapped' calculation (I trusted GitHub copilot too much!). However after fixing it, some of these numbers are still high:

| Ontology                           | Tot terms   | Tot excluded   | Tot deprecated   | Tot mappable _(!excluded, !deprecated)_   | Tot mapped _(mappable)_   | Tot unmapped _(mappable)_   | % unmapped _(mappable)_   |
|:-----------------------------------|:------------|:---------------|:-----------------|:------------------------------------------|:--------------------------|:----------------------------|:--------------------------|
| [ICD10WHO](./[unmapped_icd10who.md](http://unmapped_icd10who.md/)) | 12,542      | 0              | 0                | 12,542                                    | 18                        | 12,524                      | 99.9%                     |
| [ICD10CM](./[unmapped_icd10cm.md](http://unmapped_icd10cm.md/))   | 95,847      | 15,452         | 0                | 80,395                                    | 1,160                     | 79,235                      | 98.6%                     |
| [NCIT](./[unmapped_ncit.md](http://unmapped_ncit.md/))         | 174,278     | 155,232        | 0                | 19,046                                    | 6,734                     | 12,312                      | 64.6%                     |
| [OMIM](./[unmapped_omim.md](http://unmapped_omim.md/))         | 28,659      | 17,286         | 1,345            | 10,028                                    | 9,779                     | 249                         | 2.5%                      |
| [ORDO](./[unmapped_ordo.md](http://unmapped_ordo.md/))         | 15,238      | 6,082          | 0                | 9,156                                     | 8,934                     | 222                         | 2.4%                      |
| [DOID](./[unmapped_doid.md](http://unmapped_doid.md/))         | 13,648      | 2,599          | 2,467            | 11,048                                    | 9,748                     | 1,300                       | 11.8%                     |